python/pyspark/ml/feature.py

```diff
@@ -1250,6 +1257,20 @@ def getNumBuckets(self):
         """
         return self.getOrDefault(self.numBuckets)
 
+    @since("2.0.0")
+    def setRelativeError(self, value):
+        """
+        Sets the value of :py:attr:`relativeError`.
+        """
+        return self._set(relativeError=value)
+
+    @since("2.0.0")
+    def getRelativeError(self):
+        """
+        Gets the value of relativeError or its default value.
```

Comment:
@BryanCutler it's a good point - but I've seen that all other docs for the getters through py-ml omit the `:py:attr:`. I'm not certain if this is intended or some oversight that has become "copy-pasted" as de facto standard :) I for one would support updating all of them as it makes sense to me that the doc links match the setters. But it then needs to be done for all Python param setter methods to be consistent.

cc @jkbradley @holdenk @yanboliang